tools/test-js-with-node

```diff
@@ -298,6 +298,7 @@ EXEMPT_FILES = make_set(
         "web/src/zform.js",
         "web/src/zulip.js",
         "web/src/zulip_test.ts",
+        "web/tests/lib/make_message.js",
```

Comment:
What parts of this file don't have coverage? It is probably better to just add a small `test_make_message` test file to exercise it.